documentation/sphinx/source/api-c.rst

```diff
@@ -444,6 +444,21 @@ An |database-blurb1| Modifications to a database are performed via transactions.
    ``duration``
         If positive, the process will be first suspended for ``duration`` seconds before being rebooted.
 
+.. function:: FDBFuture* fdb_database_force_recovery_with_data_loss(FDBDatabase* database, uint8_t const* dcId, int dcId_length)
```

Comment:
I feel like there's an opportunity do add some details about what this does, e.g. what happens if there's no region dcId? Does the future completing have special significance? Is this operation atomic, or is there some wacky state you might get into if you attempt it?

I see that this is pretty much copied from the fdbcli command help text so I think it's ok as is for now

Reply:
I update some of it for what I know.